### PR TITLE
Afhs 1814 double submit automation tests

### PR DESCRIPTION
**Pull request by jbudda**
- Added a feature to test double click action on the send application button of terms and conditions page

### CODE REVIEW

File: src/test/resources/features/acceptance/decision.feature [hunk 1]
```diff
@@ -12,6 +12,13 @@ Feature: Confirm application
     And my entitlement is £9.30 per week with a first payment of £37.20
     And my claim is sent to the back end
 
+  @RequiresWiremock
+  Scenario: Double click on the send application button
+    Given I have entered my details up to the check answers page
+    When I double click send application button in terms and conditions page
```
Comment (mikeSmith08) [inline src/test/resources/features/acceptance/decision.feature]: How about `When I double the click send application button on the terms and conditions page`?

Comment (jbudda) [reply src/test/resources/features/acceptance/decision.feature]: changed as per comment @mikeSmith08

File: src/test/java/uk/gov/dhsc/htbhf/steps/TermsAndConditionsSteps.java [hunk 2]
```diff
@@ -26,6 +26,13 @@ public void doNotAcceptTermsAndConditions() {
         termsAndConditionsPage.clickContinue();
     }
 
+    @When("^I double the click send application button in terms and conditions page")
```
Comment (adeel718) [inline src/test/java/uk/gov/dhsc/htbhf/steps/TermsAndConditionsSteps.java]: Should be I double click the ...

File: src/test/java/uk/gov/dhsc/htbhf/steps/TermsAndConditionsSteps.java [hunk 3]
```diff
@@ -26,6 +26,13 @@ public void doNotAcceptTermsAndConditions() {
...
+    public void doubleClickSendApplicationButton(){
```
Comment (adeel718) [inline src/test/java/uk/gov/dhsc/htbhf/steps/TermsAndConditionsSteps.java]: Space before {

File: src/test/java/uk/gov/dhsc/htbhf/steps/TermsAndConditionsSteps.java (hunk 2 above)
Comment (mikeSmith08) [reply src/test/java/uk/gov/dhsc/htbhf/steps/TermsAndConditionsSteps.java]: Yes, it should be `I double click the send application button on the terms and conditions page`.

File: src/test/java/uk/gov/dhsc/htbhf/steps/TermsAndConditionsSteps.java (hunk 3 above)
Comment (mikeSmith08) [reply src/test/java/uk/gov/dhsc/htbhf/steps/TermsAndConditionsSteps.java]: Yes, we don't have Checkstyle or PMD running on the acceptance tests.

**Review by YetAnotherMatt — DISMISSED** [review]

**Review by dwybourn — DISMISSED** [review]

File: src/test/java/uk/gov/dhsc/htbhf/steps/TermsAndConditionsSteps.java (hunk 2 above)
Comment (jbudda) [reply src/test/java/uk/gov/dhsc/htbhf/steps/TermsAndConditionsSteps.java]: Sorry i did not understand the comment @Adeel @Mike it was "I double click the .." later changed to "I double the click .." as per above comment

Comment (jbudda) [reply src/test/java/uk/gov/dhsc/htbhf/steps/TermsAndConditionsSteps.java]: I got it thanks @mikeSmith08 . applied changes

File: src/test/java/uk/gov/dhsc/htbhf/steps/TermsAndConditionsSteps.java (hunk 3 above)
Comment (jbudda) [reply src/test/java/uk/gov/dhsc/htbhf/steps/TermsAndConditionsSteps.java]: done

**Review by AmarkumarJagtap — DISMISSED** [review]

**Review by mikeSmith08 — DISMISSED** [review]

**Review by YetAnotherMatt — DISMISSED** [review]

**Review by adeel718 — DISMISSED** [review]

**Review by YetAnotherMatt — APPROVED** [review]

File: src/test/java/uk/gov/dhsc/htbhf/utils/WireMockManagerImpl.java [hunk 4]
```diff
@@ -71,7 +71,7 @@ public void setupClaimantServiceMappingsForScenario(ClaimScenario scenario) {
 
     @Override
     public void verifyClaimantServiceRequestMatching(String expectedBody) {
```
Comment (dwybourn) [inline src/test/java/uk/gov/dhsc/htbhf/utils/WireMockManagerImpl.java]: Should we rename this method to state that only one call was made as we're now asserting that.

Comment (YetAnotherMatt) [reply src/test/java/uk/gov/dhsc/htbhf/utils/WireMockManagerImpl.java]: Good plan: `verifySingleClaimantServiceRequestMatching`?

**Review by adeel718 — APPROVED** [review]

**Pull request merged**